integration/test_integration_basics.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import sys
+import unittest
+
+from common import IpVersion
+from integration_test_base import (HttpIntegrationTestBase, HttpsIntegrationTestBase,
+                                   IntegrationTestBase)
+
+
+class TestHttp(HttpIntegrationTestBase):
+
+  def test_h1(self):
+    parsed_json = self.runNighthawk([self.getTestServerRootUri()])
+    counters = self.getNighthawkCounterMapFromJson(parsed_json)
+    self.assertEqual(counters["benchmark.http_2xx"], 25)
```

Comment:
I'd like to figure out how to reduce boilerplate even further, e.g. by adding some method that takes a map of expected key/values, counters and then iterates over the former. It can be a TODO though as this is already fairly good.

Reply:
Added